### PR TITLE
Products Response Object Updates

### PR DESCRIPTION
**Pull request by mabdullah12**
1. Image URL param addded in ProductResponse object.
2. Image URL, currency, price, and metadata param added in SKUResponse object.

### CODE REVIEW

**Comment by tpindel** [comment]
@lkulig please review PR and decide if we can merge and release.

**Comment by mabdullah12** [comment]
@tpindel @lkulig any update on this or shall I move to my customer implementation directly from the API?
Thanks!

File: src/main/java/io/voucherify/client/model/product/response/SKUResponse.java [hunk 1]
```diff
@@ -28,5 +28,14 @@
   @JsonProperty("created_at")
   private Date createdAt;
 
+  @JsonProperty("image_url")
+  private String imageUrl;
+
+  private String currency;
+
+  private String price;
```
Comment (lkulig) [inline src/main/java/io/voucherify/client/model/product/response/SKUResponse.java:36]: why `String`? it should be a number just like in ProductResponse

**Review by lkulig — CHANGES_REQUESTED** [review]

File: src/main/java/io/voucherify/client/model/product/response/SKUResponse.java (hunk 1 above)
Comment (mabdullah12) [reply src/main/java/io/voucherify/client/model/product/response/SKUResponse.java]: Hi. I have updated the price param to Long. Thanks for the correction.

**Review by lkulig — APPROVED** [review]

**Comment by mabdullah12** [comment]
@lkulig Thanks for merging it. Will the build be available on maven repository?
Thanks!

**Pull request merged**